docs/custom_installs/index.rst

```diff
@@ -1,9 +1,18 @@
 Info about custom installs
 ==========================
 
-Read the Docs is open source, which means you can run your own version of it. There are many reasons to do this, the main one being if you want a private instance. If you have to keep everything behind a firewall or VPN, this is for you.
+Read the Docs is open source, which means you can run your own version of it.
+There are many reasons to do this, the main one being if you want a private instance.
+If you have to keep everything behind a firewall or VPN, this is for you.
 
-.. warning:: Read the Docs developers do not support custom installs of our software. These documents are maintained by the community, and might not be up to date.
+If your main reasons for considering a custom installation
+are to connect to private repositories or to have fine grained access control over your documentation,
+please consider :doc:`Read the Docs for Business </commercial/index>`.
+It has those features and more!
+
+.. warning::
+    Read the Docs developers do not support custom installs of our software.
+    These documents are maintained by the community, and might not be up to date.
```

Comment:
We probably already link to our policy on open source and support of custom installs, but this could be a good place to re-iterate.

Reply:
You're talking about this, correct? https://readthedocs.com/services/#open-source-support

Reply:
I added a link to that.